<launch>

    <arg name="odom_topic" default="odom" />
    <arg name="cmd_vel_topic" default="cmd_vel" />
    <arg name="isreal" default="false" />
    <arg name="/use_sim_time" value="$(arg isreal)" />
    
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
        <!-- topic remap -->
        <remap from="odom" to="$(arg odom_topic)" />
        <remap from="cmd_vel" to="$(arg cmd_vel_topic)" />
        <rosparam file="$(find robotx_nav)/config/fake/costmap_common_params.yaml" command="load" ns="global_costmap" unless="$(arg isreal)" />
        <rosparam file="$(find robotx_nav)/config/fake/costmap_common_params.yaml" command="load" ns="local_costmap" unless="$(arg isreal)" />
        <rosparam file="$(find robotx_nav)/config/fake/local_costmap_params.yaml" command="load" unless="$(arg isreal)" />
        <rosparam file="$(find robotx_nav)/config/fake/global_costmap_params.yaml" command="load" unless="$(arg isreal)" />
        <rosparam file="$(find robotx_nav)/config/fake/base_local_planner_params.yaml" command="load" unless="$(arg isreal)" />

        <rosparam file="$(find robotx_nav)/config/wamv/costmap_common_params.yaml" command="load" ns="global_costmap" if="$(arg isreal)" />
        <rosparam file="$(find robotx_nav)/config/wamv/costmap_common_params.yaml" command="load" ns="local_costmap" if="$(arg isreal)" />
        <rosparam file="$(find robotx_nav)/config/wamv/local_costmap_params.yaml" command="load" if="$(arg isreal)" />
        <rosparam file="$(find robotx_nav)/config/wamv/global_costmap_params.yaml" command="load" if="$(arg isreal)" />
        <rosparam file="$(find robotx_nav)/config/wamv/base_local_planner_params.yaml" command="load" if="$(arg isreal)" />
    </node>

</launch>
